pkg/operator/ceph/cluster/mgr/dashboard.go

```diff
@@ -102,7 +102,8 @@ func (c *Cluster) toggleDashboardModule() error {
 }
 
 func (c *Cluster) configureDashboardModule() error {
-	hasChanged, err := client.MgrSetConfig(c.context, c.Namespace, c.cephVersion.Name, "mgr/dashboard/url_prefix", c.dashboard.UrlPrefix)
+	allMgrs := ""
+	hasChanged, err := client.MgrSetConfig(c.context, c.Namespace, allMgrs, c.cephVersion.Name, "mgr/dashboard/url_prefix", c.dashboard.UrlPrefix)
```

Comment:
`allMgrs` is empty. It is used in the `MgrSetConfig` func to build the mgr ID. Is that right?